PowerPointLabs/PowerPointLabs/TooltipsLab/Views/TooltipsLabSettingsShapeEntry.xaml.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Drawing;
+using System.IO;
+using System.Text.RegularExpressions;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Interop;
+using System.Windows.Media.Imaging;
+using Microsoft.Office.Core;
+using PowerPointLabs.ActionFramework.Common.Extension;
+using PowerPointLabs.TextCollection;
+
+namespace PowerPointLabs.TooltipsLab.Views
+{
+    /// <summary>
+    /// Interaction logic for TooltipsLabSettingsShapeEntry.xaml
+    /// </summary>
+    public partial class TooltipsLabSettingsShapeEntry : UserControl
+    {
+        private MsoAutoShapeType type;
+
+        #region Constructors
+
+        public TooltipsLabSettingsShapeEntry(MsoAutoShapeType type, Bitmap image)
+        {
+            InitializeComponent();
+            Type = type;
+            imageBox.Source = Imaging.CreateBitmapSourceFromHBitmap(
+                image.GetHbitmap(),
+                IntPtr.Zero,
+                Int32Rect.Empty,
+                BitmapSizeOptions.FromEmptyOptions());
```

Comment:
This image will have zero height and zero width?

Reply:
I think IntPtr.Zero might be offset or something, will have a comment to explain this.

Reply:
@leeyh20 After looking at how this code is used in many other labs, none of them have any comments explaining what it does. It might be a good idea to extract this block into a utility. Will be addressed in another issue.